manifests/l/Lenovo/DockManager/1.5.2.2/Lenovo.DockManager.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=QUSU.CRLF.7-4-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Lenovo.DockManager
+PackageVersion: 1.5.2.2
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /NORESTART
+  SilentWithProgress: /SILENT /NORESTART
+  InstallLocation: /DIR="<INSTALLPATH>"
+  Log: /LOG="<LOGPATH>"
+UpgradeBehavior: install
+ReleaseDate: 2024-07-30
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.lenovo.com/consumer/options/dockManagersetup_1.5.2.2.exe
+  InstallerSha256: E68D8440CD293C4A3D90032665B0F1EC1EB159BCDB6511E23E2F7F5DF978BDF2
```

```suggestion
  InstallerSha256: E68D8440CD293C4A3D90032665B0F1EC1EB159BCDB6511E23E2F7F5DF978BDF2
  ProductCode: DockManager_is1
```

Helpful metadata for better package correlation